migrations/versions/66b173ff9a64_add_shortlink_to_event.py

```diff
@@ -0,0 +1,37 @@
+"""Add shortlink to event
+
+Revision ID: 66b173ff9a64
+Revises: 94cbf12be25a
+Create Date: 2023-01-20 23:33:07.814569
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from urllib.parse import quote
+
+# revision identifiers, used by Alembic.
+revision = '66b173ff9a64'
+down_revision = '94cbf12be25a'
+branch_labels = None
+depends_on = None
+
+
+def default_shortlink(context):
+    """Generates a default shortlink out of the name of the event, by removing spaces and making it URL-safe."""
+    return quote(context.get_current_parameters()["name"].replace(" ", ""))
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    with op.batch_alter_table('event', schema=None) as batch_op:
+        batch_op.add_column(sa.Column('shortlink', sa.Text(), nullable=True, default=default_shortlink))
```

Comment:
I'd also not create the default shortlink for all past events. Let's just create  them by hand. E.g. we anyways would want "sommerakademie2023`, but probably just `soak23`, same for something like "bewerbungsworkshop"

Reply:
Yes, it's kind of an optional feature. I realize it's called shortlink, bu it's nice to have named links anyway, isn't it? No matter if they are short or not. If we want a real short link we can modify it in the admin view after all.